src/main/java/de/jetwick/snacktory/ArticleTextExtractor.java

```diff
@@ -389,24 +389,46 @@ private static String getSnippet(String data){
             return data.substring(0, 50);
     }
 
+    private static String safe_first_text(Element el) {
```

Comment:
Java convention is to use camel case for names. So the method name should be `safeFirstText` :-)

Reply:
:facepalm: ah… java. Thanks